code/modules/tgui/tgui_window.dm

```diff
@@ -321,3 +321,6 @@
 			// Resend the assets
 			for(var/asset in sent_assets)
 				send_asset(asset)
+
+/datum/tgui_window/vv_edit_var(var_name, var_value)
+	return var_name != NAMEOF(src, id) && ..()
```

Comment:
?

Reply:
I couldn't find a directly exploitable use for varediting id but there were some close calls (i.e modifying output type). This is just caution